src/main/cpp/Robot.cpp

```diff
@@ -235,6 +235,38 @@ m_drive.ArcadeDrive(throttleExp, turnInput);
 }
 }
 //Fire!
+
+double Robot::LimelightDistance(){
+
+  //Yet another limelight initalization block
+  nt::NetworkTableEntry txEntry;
+  nt::NetworkTableEntry tyEntry;
+  nt::NetworkTableEntry taEntry;
+
+  double ta, tx, ty;
+
+  auto inst = nt::NetworkTableInstance::GetDefault();
+  auto table = inst.GetTable("limelight-bepis");
+  txEntry = table->GetEntry("tx");
+  tyEntry = table->GetEntry("ty");
+  taEntry = table->GetEntry("ta");
+
+
+  txEntry.SetDouble(tx);
+  tyEntry.SetDouble(ty);
+  taEntry.SetDouble(ta);
+
+  //Map tyEntry (0-1) to Limelight FOV (0 - 49.7 deg)
+
+  //How the [redacted by Eric] do I spell theta?
+  double theta = (49.7*ty)+36.711;
```

Comment:
May be good to put a few comments in here saying how you got these constants

Reply:
That seems like a good idea. I'll do that during lunch.